datacite-transform/build.gradle

```diff
@@ -1,41 +1,80 @@
-import java.nio.file.Files
-import java.nio.file.Paths
-
 plugins {
     id 'java'
-    id "org.unbroken-dome.xjc" version "2.0.0"
```

Comment:
This was last updated four years ago, a PR was created by a third party for jakarta update in recent time, but nothing has been commented or merged in four years.